com.ca.lsp.cobol/lsp-service-cobol/src/test/java/com/ca/lsp/cobol/usecases/TestReplacingPseudoTextReplacesLevelNumber.java

```diff
@@ -34,7 +34,7 @@
           + "2      DATA DIVISION.\n"
           + "3      WORKING-STORAGE SECTION.\n"
           + "4      01  {$*PARENT}.\n"
-          + "5      COPY {~REPL} REPLACING == 01 AA-PARENT == BY == 05  EE-PARENT ==\n"
+          + "5      COPY {~REPL} REPLACING == 01 AA-PARENT == by == 05  EE-PARENT ==\n"
```

Comment:
why only this "by" was made lowercase?

Reply:
Here and later it is just to assert case-insensitivity. This exact test class was chosen randomly